### PR TITLE
proposes a process for highlighting code experiments

### PR DESCRIPTION
**Pull request by ultrasaurus**
This was an Action Item from Serverless WG meeting.

Signed-off-by: Sarah Allen sarahallen@google.com

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@duglin my commit had a sign-off, not sure what I did wrong there...

**Comment by duglin** [comment]
hmm not sure but did you add it manually? I think it might not like that its missing the `<>`.
e.g. `Signed-off-by: John Doe <johndoe@exmaple.com>`

File: community/demos.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ If you have a demo of CloudEvents in action, please add a link here.
+ If there isn't an associated blog or github repo, feel free to add
+ descriptive text as a markdown file in community/demos/
```
Comment (duglin) [inline community/demos.md]: Missing trailing period.

File: community/open-source.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+If you have created (or know of) an open source library or product that
+uses CloudEvents, please include in the list below
```
Comment (duglin) [inline community/open-source.md]: missing trailing period or colon?

**Comment by duglin** [comment]
@ultrasaurus overall looks good. Can we get a link to the community/README.md file from the top-level README, otherwise people may not find this info as easily?

**Comment by duglin** [comment]
href checker isn't happy. I think you need to add the ".md" extension

**Comment by duglin** [comment]
LGTM

**Comment by clemensv** [comment]
LGTM

**Comment by duglin** [comment]
@ultrasaurus can you rebase?

**Comment by duglin** [comment]
Sarah - can you rebase again? Then I can merge.

Approved on 4/26 call.

**Comment by duglin** [comment]
@ultrasaurus I rebased for you - hope that's ok.

**Comment by duglin** [comment]
Approved on 4/26 call

**Comment by ultrasaurus** [comment]
I thought I had rebased.  Thanks for taking care of that @duglin !

**Pull request merged**